ChagokChagok/ChagokChagok/Helper/FavoriteButton.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  FavoriteButton.swift
+//  ChagokChagok
+//
+//  Created by Youngwoong Choi on 2022/06/13.
+//
+
+import SwiftUI
+
+struct FavoriteButton: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    
+    @FetchRequest(entity: Pin.entity(), sortDescriptors: [NSSortDescriptor(keyPath: \Pin.date, ascending: true)],
+        animation: .default) private var pins: FetchedResults<Pin>
+    
+    var pin = Pin()
+    
+    var body: some View {
+        Button {
+            if pin.isFavorite == true {
+                pin.isFavorite.toggle()
+                pin.isFavorite = false
+            } else {
+                pin.isFavorite.toggle()
+                pin.isFavorite = true
+            }
```

Comment:
```suggestion
            pin.isFavorite.toggle()
```

동일한 처리를 두번씩 진행되고 있는 것 같아요 .. !

Reply:
그냥 토글 하나만 남겨도 되겠죠...?